pom.xml

```diff
@@ -36,6 +36,8 @@
 
     <properties>
         <version.keycloak>${project.version}</version.keycloak>
+        <version.keycloak.client>26.0.0-SNAPSHOT</version.keycloak.client>
```

Comment:
Why are we using a `26.0.0-SNAPSHOT` version if other repos use the placeholder `999.0.0-SNAPSHOT`? Not saying we should not use 26, but just wondering why we want to use a different convention here.

Reply:
AFAIK we use `26.0.0-SNAPSHOT` just temporarily to not clash with existing snapshots. 

For example here is already "999" snapshot for policy-enforcer https://s01.oss.sonatype.org/content/repositories/snapshots/org/keycloak/keycloak-policy-enforcer/ , which is available due the snapshot build before this commit in keycloak main: https://github.com/keycloak/keycloak/commit/5526976d1cbf3204f6a2a1ddf189c140d08f47e9 . So now when you setup sonatype snapshot for keycloak-client repository, there will be new "26.0.0" snapshot and it will be clear that you did your sonatype snapshot setup correctly and it is the build from `keycloak-client` repository rather than some older snapshot build from `keycloak` repository :-)

But I believe that we can later update that also to `999.0.0-SNAPSHOT` after some time? @pskopek Do you have any more insight to this?